.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -5,35 +5,39 @@ about: Create a report to help us improve
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+### Describe the bug
+<!-- A clear and concise description of what the bug is. Please be as descriptive as possible. -->
 
-**To Reproduce**
-Steps to reproduce the behavior:
+### To Reproduce
+<!-- Describe the steps to reproduce the behavior.-->
 
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
+1. Go to '…'
+2. Click on '…'
+3. Scroll down to …'
 4. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+### Actual behavior:
```

Comment:
This is the only piece of this PR I feel on the fence about.  I feel it's redundant when a bug is described above and more often than not we're left wondering what someone expected to happen.